### PR TITLE
Add script(s) for issue 3226

### PR DESCRIPTION
**Pull request by aderiabin**
ATF Test Scripts to check #[3226]

This PR is **ready** for review.

### Summary
Scripts to check 3226 issue

### ATF version
develop

### Changelog
 - Added scripts to check 3226 issue

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+-- Description: SDL crashes when the HMI sends UI.OnTouchEvent
+--   with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- Steps :
+-- 1. Navi_app is registered and activated
+-- 2. OnTouchEventOnlyGroup is allowed by policies
+-- 3. HMI sends UI.OnTouchEvent with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- SDL does:
+--  - SDL does not transfer OnTouchEvent notification to the application
```
Comment (VjKlepikov) [inline test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua:13]:
```suggestion
--  - not transfer OnTouchEvent notification to the application
```

**Review by VjKlepikov — APPROVED** [review]

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua (hunk 1 above)
Comment (VjKlepikov) [reply test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua:13]: @aderiabin Please add test to  test_sets/Defects/Defects_release_8_0.txt

**Comment by VjKlepikov** [comment]
@aderiabin Please add test to test_sets/Defects/Defects_release_8_0.txt

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua (hunk 1 above)
Comment (aderiabin) [reply test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua]: See b6f3a19

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, init HMI, connect Mobile", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("PolicyTableUpdate with HMI types", common.policyTableUpdate, { ptUpdate })
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
```
[the remark this answers is not in the document]
Comment (aderiabin) [reply test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua]: See b6f3a19

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+-- 3. HMI sends UI.OnTouchEvent with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- SDL does:
+--  - SDL does not transfer OnTouchEvent notification to the application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
```
[the remark this answers is not in the document]
Comment (aderiabin) [reply test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua]: See b6f3a19

File: test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/3226
+---------------------------------------------------------------------------------------------------
+-- Description: SDL crashes when the HMI sends UI.OnTouchEvent
+--   with the 'ts' param has a value greater than maxvalue (not valid)
+--
+-- Steps :
```
[the remark this answers is not in the document]
Comment (aderiabin) [reply test_scripts/Defects/8_0/3226_SDL_crashes_if_HMI_sends_UI_OnTouchEvent_and_ts_has_value_greater_than_maxvalue.lua]: See b6f3a19

**Comment by aderiabin** [comment]
@VjKlepikov @VNikiforov-dev please take a look at b6f3a19 about your review comments

**Review by VNikiforov-dev — APPROVED** [review]

**Review by dboltovskyi — APPROVED** [review]

**Comment by aderiabin** [comment]
It was replaced `User story` by `Issue` in description.

**Comment by aderiabin** [comment]
Created PR to smartdevicelink repository (https://github.com/smartdevicelink/sdl_atf_test_scripts)

**Pull request closed without merging**